h/views/status.py

```diff
@@ -9,9 +9,13 @@
 
 @json_view(route_name="status")
 def status(request):
+    request.response.headers["Cache-Control"] = "no-store"
+
```

Comment:
I think you can just do `@json_view(..., http_cache=0)`. Assuming our custom `@json_view` decorator passes `http_cache` though to Pyramid's `view_config`. See `http_cache` in the non-predicate `view_config` args: https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html#nonpredicate-view-args

Also, `Response` objects have convenience attrs for all the common response headers. E.g. I think you can do `request.response.cache_control = "no-store"`. See https://docs.pylonsproject.org/projects/pyramid/en/latest/api/interfaces.html?highlight=cache_expires#pyramid.interfaces.IResponse.cache_control But `http_cache` in the `@view_config` does this for you